contracts/pcv/convex/VotiumBriber.sol

```diff
@@ -43,7 +43,7 @@ contract VotiumBriber is CoreRef {
     /// the _proposal ID, if _choiceIndex is out of range, or of block.timestamp
     /// is after the deadline for bribing (usually 6 hours before Convex snapshot
     /// vote ends).
-    function bribe(bytes32 _proposal, uint256 _choiceIndex) public onlyGovernorOrAdmin whenNotPaused {
+    function bribe(bytes32 _proposal, uint256 _choiceIndex) public onlyTribeRole(TribeRoles.VOTIUM_ROLE) whenNotPaused {
```

Comment:
should this be `oneOfTwoRoles(TribeRoles.GOVERNOR, TribeRoles.VOTIUM_ROLE)` or was removing the gov intentional?

Reply:
removing gov is fine, its only ever been called by the ops timelock